src/app/views/band-albums.html

```diff
@@ -1,5 +1,51 @@
-<div class="band-detail-container">
-    <img src="https://i.scdn.co/image/66302ca80395b6be3600d5f0ef69db9e0c43f4f5" alt="">
-    <img src="https://i.scdn.co/image/e2b11f82ba8d5aecc6486fe28225db43a003caa1" alt="">
-    <img src="https://i.scdn.co/image/3a299e917835b053f53441280262f10829bd1c8e" alt="">
+<div class="heading">
+	<a href="/#"><img class="header-icon" src="http://www.iconarchive.com/download/i85639/graphicloads/100-flat/zoom-search-2.ico" alt="Spotify search icon"></a>
+	<div class="search" ng-controller="viewCtrl">
+        <i class="material-icons">search</i> 
+        <input ng-model="stringSearch"  ng-change="viewCtrl.changeView('/results')" type=search class="input-search" placeholder="Type the name of your favorite artist">
+    </div>
+</div>
+
+<div class="main" ng-controller="artistCtrl" >
```

Comment:
you can register the controller associated to the template directly in each routing entry